### DISCUSSION TITLE
meaning of "exit" : true / false in config.json

### DISCUSSION THREAD

**Discussion by Cyberbastler** [UnchartedBull/OctoDash#1316] [Q&A] [answered]
Hi all,
I am configuring custom actions and can't find any explanation what the "exit" does or doesn't.
Did I miss something?

**Comment by UnchartedBull** [comment, marked as answer]
It means that OctoDash will return back to the HomeScreen once the action has been sent, it is formulated a little clearer in the settings UI :)

**Reply by Cyberbastler** [reply]
Thx a lot!
